test/SBT.t.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import "forge-std/Test.sol";
+import {SevenTeenTwentyNineProofOfContribution} from "../src/SevenTeenTwentyNineProofOfContribution.sol";
+
+contract SBTTest is Test {
+    //
+    SevenTeenTwentyNineProofOfContribution sbt;
+
+    function setUp() public {
+        sbt = new SevenTeenTwentyNineProofOfContribution();
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                        Issuing
+    //////////////////////////////////////////////////////////////*/
+
+    // TODO decide if we should enforce 1 credential of each type per user or support multiple / max
+    // if yes — replace amount of 1 in SBT with something other than a magic number
+    // if no — add to functions and events, consider adding an immutable constructor arg for max
+
+    // TODO decide if we should support ownership transfer in SBT constructor, or leave to implementers
```

Comment:
Suggestion: do it in the constructor OR provide a concrete implementation that does it in its constructor

Reply:
FWIW OZ Ownable already does this, for the deploying EOA. I will add to our 1729 Writers contract, not the base SBT contract.